[2024-04-18T02:48:07.066+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-18T02:48:07.145+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_sensors.failure_timeout_sensor_python manual__2024-04-18T02:47:33.787232+00:00 [queued]>
[2024-04-18T02:48:07.184+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_sensors.failure_timeout_sensor_python manual__2024-04-18T02:47:33.787232+00:00 [queued]>
[2024-04-18T02:48:07.186+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-18T02:48:07.269+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonSensor): failure_timeout_sensor_python> on 2024-04-18 02:47:33.787232+00:00
[2024-04-18T02:48:07.285+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1585) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-18T02:48:07.286+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_sensors', 'failure_timeout_sensor_python', 'manual__2024-04-18T02:47:33.787232+00:00', '--job-id', '22', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_sensors.py', '--cfg-path', '/tmp/tmphpvikogi']
[2024-04-18T02:48:07.290+0000] {standard_task_runner.py:63} INFO - Started process 1619 to run task
[2024-04-18T02:48:07.290+0000] {standard_task_runner.py:91} INFO - Job 22: Subtask failure_timeout_sensor_python
[2024-04-18T02:48:07.398+0000] {task_command.py:426} INFO - Running <TaskInstance: example_sensors.failure_timeout_sensor_python manual__2024-04-18T02:47:33.787232+00:00 [running]> on host f31c3e7472ea
[2024-04-18T02:48:07.697+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_sensors' AIRFLOW_CTX_TASK_ID='failure_timeout_sensor_python' AIRFLOW_CTX_EXECUTION_DATE='2024-04-18T02:47:33.787232+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-18T02:47:33.787232+00:00'
[2024-04-18T02:48:07.700+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-18T02:48:07.702+0000] {python.py:75} INFO - Poking callable: <function failure_callable at 0x7f151a1cae80>
[2024-04-18T02:49:07.702+0000] {python.py:75} INFO - Poking callable: <function failure_callable at 0x7f151a1cae80>
[2024-04-18T02:49:07.705+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-18T02:49:07.712+0000] {taskinstance.py:2502} INFO - Sensor has timed out; run duration of 60.004020884000056 seconds exceeds the specified timeout of 3.0.
[2024-04-18T02:49:07.746+0000] {taskinstance.py:1205} INFO - Marking task as SKIPPED. dag_id=example_sensors, task_id=failure_timeout_sensor_python, execution_date=20240418T024733, start_date=20240418T024807, end_date=20240418T024907
[2024-04-18T02:49:07.799+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-18T02:49:07.839+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-18T02:49:07.843+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
